elide-datastore/testng-audit.xml

```diff
@@ -5,10 +5,10 @@
   ~ See LICENSE file in project root for terms.
   -->
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
-<suite name="Suite">
+<suite name="Audit Test Suite">
     <test name="com.yahoo.elide.tests.AuditIT">
         <classes>
```

Comment:
There is only 1 class in this package. But, I'll change this to packages instead to pick up all the classes, just in case somebody adds more classes in this package and forgets to update this